inc/usb_ccid.h

```diff
@@ -0,0 +1,71 @@
+#include <stdint.h>
+
+#ifndef _USB_CCID_H_
+#define _USB_CCID_H_
+#ifdef __cplusplus
+    extern "C" {
+#endif
+
+#define USB_CLASS_CCID                   0x0b    /**<\brief Smart Card Device class */
+#define USB_DTYPE_FUNCTIONAL_DESC   0x21   /**<\brief Funcitonal descriptor.*/
+#define USB_DTYPE_CCID_FUNCTIONAL   0x21   /**<\brief Functional descriptor.*/
+
+/** Endpoint size in bytes of the CCID data being sent between IN and OUT endpoints. */
+#define CCID_EPSIZE          64
+
+
+
+#define CCID_CURRENT_SPEC_RELEASE_NUMBER            0x0110
+#define CCID_VOLTAGESUPPORT_5V                      0
+#define CCID_VOLTAGESUPPORT_3V                      (1 << 0)
+#define CCID_VOLTAGESUPPORT_1V8                     (1 << 1)
+
+#define CCID_PROTOCOL_T0                            0
+#define CCID_PROTOCOL_T1                            (1 << 0)
+
+#define CCID_ICCSTATUS_PRESENTANDACTIVE             0
+#define CCID_ICCSTATUS_PRESENTANDINACTIVE           (1 << 0)
+#define CCID_ICCSTATUS_NOICCPRESENT                 (1 << 1)
+
+#define CCID_COMMANDSTATUS_PROCESSEDWITHOUTERROR    0
+#define CCID_COMMANDSTATUS_FAILED                   (1 << 6)
+#define CCID_COMMANDSTATUS_TIMEEXTENSIONREQUESTED   (1 << 7)
+
+#define CCID_ERROR_NOERROR          0
+#define CCID_ERROR_SLOTNOTFOUND     5
+
+#define CCID_ABORT                                  0x1
+#define CCID_GET_CLOCK_FREQUENCIES                  0x2
+#define CCID_GET_DATA_RATES                         0x3
+
+struct usb_ccid_descriptor {
```

Comment:
For me, it is best to use field names that are defined in the specification to achieve more consistency.